pom.xml

```diff
@@ -5,7 +5,7 @@
 
   <groupId>com.neverpile</groupId>
   <artifactId>url-crypto-kit</artifactId>
-  <version>${revision}</version>
+  <version>1.0-SNAPSHOT</version>
```

Comment:
Absicht?

Reply:
Yes, since this represents the current pattern we use in other projects as well.